Doc/c-api/set.rst

```diff
@@ -163,4 +163,6 @@ subtypes but not for instances of :class:`frozenset` or its subtypes.
 
 .. c:function:: int PySet_Clear(PyObject *set)
 
-   Empty an existing set of all elements.
+   Empty an existing set of all elements. Return ``0`` on
+   success or ``-1`` on failure. Raise :exc:`SystemError` if *set* is not an
+   instance of :class:`set` or its subtype.
```

Comment:
This doc is quite generic and doesn't explain that it cannot fail if you pass a set type or a subtype of the set type.

It only fails the argument type is not set.

Reply:
Done, should be better now!